libbeat/outputs/util.go

```diff
@@ -17,16 +17,49 @@
 
 package outputs
 
+import (
+	"fmt"
+
+	"github.com/elastic/beats/v7/libbeat/publisher"
+	"github.com/elastic/beats/v7/libbeat/publisher/queue"
+	"github.com/elastic/beats/v7/libbeat/publisher/queue/diskqueue"
+	"github.com/elastic/beats/v7/libbeat/publisher/queue/memqueue"
+	"github.com/elastic/elastic-agent-libs/config"
+)
+
 // Fail helper can be used by output factories, to create a failure response when
 // loading an output must return an error.
 func Fail(err error) (Group, error) { return Group{}, err }
 
 // Success create a valid output Group response for a set of client instances.
-func Success(batchSize, retry int, clients ...Client) (Group, error) {
+func Success(cfg config.Namespace, batchSize, retry int, clients ...Client) (Group, error) {
```

Comment:
Let's document here that `cfg` is expected to contain queue configuration, since its datatype here `config.Namespace` doesn't quite make that obvious.

Reply:
Same for `SuccessNet` below.